### PR TITLE
Added utility routes to docs.

### PR DESCRIPTION
**Pull request by nathanialhenniges**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] Docs have been added/updated (for bug fixes / features) wherever necessary

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation content changes

## What is the current behavior?

Issue Number: [LINSTA-643](https://scopeweb.atlassian.net/browse/LINSTA-643)

## Other information

### CODE REVIEW

File: source/index.html.md [hunk 1]
```diff
@@ -2628,3 +2628,82 @@ This endpoint shows all the leads in a array.
...
+  -H "Authorization: Bearer jsonwebtoken" \
+  -H 'Content-Type: application/x-www-form-urlencoded' \
+  --data-urlencode 'description=This is great.' \
+  --data-urlencode 'feedbackType=Good' 
+```
+
+>The above command returns JSON structured like this:
+
+```json
+{ "message": "Feedback sent.", "status": 200 }
```
Comment (stmoerman) [inline source/index.html.md:2695]: Just to double check, is this the response we are getting here? If so, we need to make a ticket for updating that.

**Review by stmoerman — CHANGES_REQUESTED** [review]
Not sure if it needs to be changed on the API or in the documentation, just curious about a response.

File: source/index.html.md (hunk 1 above)
Comment (nathanialhenniges) [reply source/index.html.md:2695]: That is the current response in the code.

Comment (stmoerman) [reply source/index.html.md:2695]: Update it in the API documentation to "Uw feedback is verstuurd"

**Review by stmoerman — CHANGES_REQUESTED** [review]
Small change for the feedback message.

**Review by stmoerman — APPROVED** [review]
👍

**Pull request merged**